packages/editor-ui/src/utils.ts

```diff
@@ -1,6 +1,35 @@
 import xss, { friendlyAttrValue } from 'xss';
 import { Primitives, Optional, JsonSchema, JsonSchemaType } from "@/Interface";
 
+// Holds weird date formats that we encounter when working with strings
+// Should be extended as new cases are found
+const CUSTOM_DATE_FORMATS = [
+	/\d{1,2}-\d{1,2}-\d{4}/, // Should handle dash separated dates with year at the end
+	/\d{1,2}\.\d{1,2}\.\d{4}/, // Should handle comma separated dates
```

Comment:
Could be just one regex and also include slash `/\d{1,2}\[.-/]\d{1,2}\[.-/]\d{4}/`

Reply:
Was thinking about a single regex but my logic was that this can be easily expanded and it's also a bit readable. Dates with slashes are in the test case and are passing, what format exactly are you thinking of?

Reply:
Yes, that could work also. I just like to think always about the performance but in this case, it's hard to tell if it is better to iterate through smaller regexes or just have one that covers all

Regex is always hard to read but as you become more experienced in it your eyes will immediately split it into chunks you can recognize, parentheses for the matching groups, brackets for characters etc...

For me, the merged regex is easier to read because as I start reading I'm already in scope for the parts whereas if I have to read to separate regexes (even if they're very similar) I can start interpreting it from the beginning again